src/articles/_posts/2009-03-06-javascript-best-practices.md

```diff
@@ -522,8 +522,8 @@ The other problem of nesting is variable names and loops. As you normally start
 
 	function renderProfiles(o) {
 		var out = document.getElementById('profiles');
+		var ul = document.createElement('ul');
```

Comment:
Moved assignment of ul out of loop so not reassigned for every element accessed.